services/resource-usage-tracker/tests/unit/with_dbs/test_api_resource_tracker__list.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime, timezone
+from typing import Any, Final, Iterator
+from unittest import mock
+
+import faker
+import httpx
+import pytest
+import sqlalchemy as sa
+from pytest_mock import MockerFixture
+from simcore_postgres_database.models.resource_tracker import resource_tracker_container
+from starlette import status
+from yarl import URL
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+FAKE: Final = faker.Faker()
```

Comment:
you do not need to have a global here. there is also a faker fixture

Reply:
i will add it in the upcoming PR today afternoon/tommorow